libstatistics_collector/include/libstatistics_collector/collector/collector.hpp

```diff
@@ -12,19 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef SYSTEM_METRICS_COLLECTOR__COLLECTOR_HPP_
-#define SYSTEM_METRICS_COLLECTOR__COLLECTOR_HPP_
+#ifndef LIBSTATISTICS_COLLECTOR__COLLECTOR__COLLECTOR_HPP_
+#define LIBSTATISTICS_COLLECTOR__COLLECTOR__COLLECTOR_HPP_
 
 #include <mutex>
 #include <string>
 
 #include "metric_details_interface.hpp"
-#include "moving_average_statistics/moving_average.hpp"
-#include "moving_average_statistics/types.hpp"
+#include "libstatistics_collector/moving_average_statistics/moving_average.hpp"
+#include "libstatistics_collector/moving_average_statistics/types.hpp"
 
 #include "rcpputils/thread_safety_annotations.hpp"
 
-namespace system_metrics_collector
+namespace libstatistics_collector
```

Comment:
Given the dir stucture shouldn't the namespace be 

namespace libstatistics_collector {
namespace collector {

Reply:
+1

Reply:
Done.

Reply:
Could you remove the other one, the goal is to have a 1:1 mapping between the directory structure, and the namespace?

Reply:
We're currently at 1:1 mapping between the namespace and the directory structure. Thanks!